changelogs/fragments/3330-bugfix-keycloak-authentication-flow-requirements-not-set-correctly.yml.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - keycloak_authentication - fix bug, the requirement was always on DISABLED when creating a new authentication flow
```

Is `DISABLED` a value?
```suggestion
  - keycloak_authentication - fix bug, the requirement was always on ``DISABLED`` when creating a new authentication flow
```